### ISSUE TITLE
POST /apis returns no identifying information, making later deprovision impossible

### ISSUE THREAD

**Issue by drauschenbach** [motiv-labs/janus#97]
**Reproduction Steps:**

1. Register an upstream via `POST /apis`
2. Observe Janus responds with an empty body, no Location header, and no other headers containing any identifying information.
3. Try deleting the API with the `slug` name, whatever that is, and observe that also does not work.

**Expected behavior:**

`POST /apis` returns a `Location: /apis/<assigned-mongo-_id>` header.

**Observed behavior:**

`POST /apis` returns no identifying information in its headers, and an empty body. An operator has to bypass the self-service REST API and peek into the Mongo DB, find the API registration, and then obtain the `_id` field value which can then be used in a delete operation:

```bash
$ curl -i -X DELETE -H "Authorization: Bearer eyJ..." http://localhost:8080/apis/58b8521...

HTTP/1.1 204 No Content
```

**Janus version:** [1.7.4-rc.6](https://github.com/hellofresh/janus/releases/tag/1.7.4-rc.6)

**Comment by italolelis** [comment]
Hey @drauschenbach,

True, I did a poor job on returning useful information after registering the API. 
I will try to implement this as soon as possible.

**Comment by italolelis** [comment]
solved  in #147
